config/settings/base.py

```diff
@@ -601,13 +601,13 @@
 # -------------------------------------------------------------------------------
 EP_API_URL = env(
     "EP_API_URL",
-    default="https://djzdv19waj3wh.cloudfront.net/api/v1/search",
+    default="EP_API_URL",
```

Comment:
put some dummy value, in order to understand the issue, if it comes
like https://ep_api.url

Reply:
done